### DISCUSSION TITLE
Yolov5: recommended imgsz

### DISCUSSION THREAD

**Discussion by igor-rud-kr** [AxisCommunications/axis-model-zoo#43] [Q&A]
What is recommended imgsz for model re-training?

**Comment by Corallo** [comment]
Hi @igor-rud-kr 

That depends on the speed/accuracy performances you are looking for. The larger and slower the model gets.
One important thing to take into account is that it is best if the image size used in the training follows the same aspect ratio that you will get when you use your model on the device.
